### ISSUE TITLE
Loss Function add

### ISSUE THREAD

**Issue by arucon** [farizrahman4u/keras-contrib#109]
Hello

I want to add loss function of Sigma RMSE
I'm using this for fitting series of weather data(Temp, WindSpeed, etc)
Used it by custom Loss Function, but It can be useful to other people

Actually tried to PR to Keras(https://github.com/fchollet/keras/pull/6996)
fchollet sent me here

After I read CONTRIBUTING.md, I'm writing this issue for PR

Thanks.

**Comment by the-moliver** [comment]
Do you have a reference for this? Why is there a sum on axis -1 after you already did a mean on axis -1? How is this different than RMSE?

**Comment by arucon** [comment]
You are right.
Sorry 
Can you close it?
